docs/transpile/rest-api-transpilation.mdx

```diff
@@ -0,0 +1,116 @@
+
+* You can either transpile [locally via Qiskit](https://docs.quantum.ibm.com/transpile/defaults-and-configuration-options) before generating the QASM string, or:
+
+* Use the [Cloud Transpiler API](https://cloud-transpiler.quantum.ibm.com/docs)
+
+<Admonition type="note">
+    [Transpilation is necessary](https://docs.quantum.ibm.com/announcements/product-updates/2024-02-14-qiskit-runtime-primitives-update) prior to submitting a circuit to IBM Quantum backends. 
+</Admonition>
+
+
+### Query the Cloud Transpilation service
+
+Query the [Cloud Transpilation service](https://cloud-transpiler.quantum-computing.ibm.com/docs) and provide your QASM string as input. See more details and documentation [here](https://docs.quantum.ibm.com/api/qiskit-transpiler-service-rest). 
+
+<Admonition type="note">
+    This experimental service is only available for IBM Quantum Premium Plan users.
+</Admonition>
+
+<Tabs>
+    <TabItem value="Qiskit Transpiler" label="Qiskit Transpiler">
+        ```python
+backend='ibm_brisbane' #choose your IBM Backend
+
+headers = {
+    "accept": "application/json",
+    "Authorization": f"Bearer {token}",
+    "Content-Type": "application/json",
+        }
+body= {
+    "qasm_circuits": qasm_string,
+}
+params = {
+        "backend": backend,
+        "optimization_level": 1,
+        "ai": "false", # "true", "false" and "auto"
+        }
+
+resp = requests.post(
+            "https://cloud-transpiler.quantum.ibm.com/transpile",
+            headers=headers,
+            json=body,
+            params=params,
+        )
+        ```
+    </TabItem>
+
+    <TabItem value="AI Transpiler" label="AI Transpiler">
+        ```python
+backend='ibm_brisbane' #choose your IBM Backend
+
+headers = {
+    "accept": "application/json",
+    "Authorization": f"Bearer {token}",
+    "Content-Type": "application/json",
+        }
+body= {
+    "qasm_circuits": qasm_string,
+}
+params = {
+        "backend": backend,
+        "optimization_level": 1,
+        "ai": "true", # "true", "false" and "auto"
+        "coupling_map": [],
+        "qiskit_transpile_options": [],
+        "ai_layout_mode": 'OPTIMIZE', # 'KEEP', 'OPTIMIZE', 'IMPROVE'
+        }
+
+resp = requests.post(
+            "https://cloud-transpiler.quantum.ibm.com/transpile",
+            headers=headers,
+            json=body,
+            params=params,
+        )
+        ```
+    </TabItem>
+</Tabs>
+
+
+<Admonition type="note">
+    Since there may be cases where it’s better to forego the use of AI-powered passes, we’ve developed a third option for the ai parameter that gives users the best of both worlds. Set the ai parameter to `ai="auto"`, and this will enable the system to decide automatically whether to apply the standard Qiskit heuristic passes or the new AI-powered passes based on the particulars of your circuit.
```

Comment:
```suggestion
    Since there may be cases where it’s better to forego the use of AI enhanced passes, we’ve developed a third option for the ai parameter that gives users the best of both worlds. Set the `ai` parameter to `ai="auto"`, and this will enable the system to decide automatically whether to apply the standard Qiskit heuristic passes or the new AI-powered passes based on the particulars of your circuit.
```